package.json

```diff
@@ -39,10 +39,11 @@
   "dependencies": {
     "ajv": "^8.0.0",
     "compare-versions": "^5.0.0",
+    "ethereum-cryptography": "^2.0.0",
```

Comment:
I'd use ethers instead (because I think we should reduce the number of different library we use, and need to understand). It may be a bit bigger, but we can possibly only pull a small subpackage.